opal/group.go

```diff
@@ -574,6 +586,21 @@ func resourceGroupRead(ctx context.Context, d *schema.ResourceData, m any) diag.
 	}
 	d.Set("reviewer_stage", reviewerStagesI)
 
+	groupResources, _, err := client.GroupsApi.GetGroupResources(ctx, group.GroupId).Execute()
+	if err != nil {
+		return diagFromErr(ctx, err)
+	}
+
+	groupResourcesI := make([]any, 0, len(reviewerStages))
```

Comment:
shouldn't this be `make([]any, 0, len(groupResources.GroupResources))`?

Reply:
thanks